app/V1Module/presenters/CommentsPresenter.php

```diff
@@ -97,10 +118,19 @@ public function actionAddComment(string $id) {
     $isPrivate = filter_var($this->getRequest()->getPost("isPrivate"), FILTER_VALIDATE_BOOLEAN);
     $comment = Comment::createComment($thread, $user, $text, $isPrivate);
 
-    $this->comments->persistComment($comment);
-    $this->comments->persistThread($thread);
+    $this->comments->persist($comment, false);
+    $this->comments->persist($thread, false);
     $this->comments->flush();
 
+    // send email to all participants in comment thread
+    if ($solution = $this->assignmentSolutions->get($id)) {
```

Comment:
It would be nice somehow emphasize that `=` inside is actually an assignment. I usually apply additional () around the assignment expression.

Reply:
Like `if (($solution = $this->assignmentSolutions->get($id))) {}` ?